### PR TITLE
Feat/AZCWSB 42 derive sub account

### PR DESCRIPTION
**Pull request by Nokiusz**
this pr consists of:
- changing paths to correct ones
- redesigning derivation flow
- adjusted tests
-added translations and assets

### CODE REVIEW

File: packages/extension-ui/src/Popup/Derive/index.tsx [hunk 1]
```diff
@@ -36,12 +36,20 @@ function Derive({ isLocked }: Props): React.ReactElement<Props> {
...
     () => accounts.find((a) => a.address === parentAddress)?.genesisHash || null,
     [accounts, parentAddress]
   );
 
+  const parentIsExternal = useMemo(
+    () => accounts.find((a) => a.address === parentAddress)?.isExternal || null,
+    [accounts, parentAddress]
+  );
+
+  const isExternalResult = parentIsExternal || 'false';
```
Comment (jalooc) [inline packages/extension-ui/src/Popup/Derive/index.tsx:51]: Won't simply:
```suggestion
  const parentIsExternal = useMemo(
    () => accounts.find((a) => a.address === parentAddress)?.isExternal || 'false',
    [accounts, parentAddress]
  );
```
achieve the same effect?

File: packages/extension-ui/src/components/AccountNamePasswordCreation.tsx [hunk 2]
```diff
@@ -126,20 +135,34 @@ function AccountNamePasswordCreation({
...
+            />
+          )}
         </div>
       </ScrollWrapper>
       <VerticalSpace />
       {onBackClick && buttonLabel && (
-        <ButtonArea footer={footer}>
-          <BackButton onClick={_onBackClick} />
+        <ButtonArea footer={!isDeriving && footer}>
+          {!!master && isDeriving ? (
```
Comment (jalooc) [inline packages/extension-ui/src/components/AccountNamePasswordCreation.tsx:156]: You don't need casting when using ternary operator.
```suggestion
          {master && isDeriving ? (
```

File: packages/extension-ui/src/Popup/Derive/DerivationPath.tsx [hunk 3]
```diff
@@ -21,7 +19,24 @@ interface Props extends ThemeProps{
...
+max-width: 284px;
+gap: 4px;
+position: relative;
+margin-bottom: 4px;
+
+label {
+color: ${({ isLocked, theme }: StyledInputWithLabelProps) => (isLocked ? theme.disabledTextColor : theme.subTextColor)};
+opacity: 1;
+
+}
```
Comment (plakak) [inline packages/extension-ui/src/Popup/Derive/DerivationPath.tsx:36]: 2 spaces of indent would be nice here

File: packages/extension-ui/src/Popup/Derive/SelectParent.tsx [hunk 4]
```diff
@@ -123,6 +141,30 @@ export default function SelectParent({
...
 
+  const goTo = useCallback((path: string) => () => onAction(path), [onAction]);
+
+  const [isPasswordVisible, setIsPasswordVisible] = useState(false);
+
+  const _handleInputTypeChange = useCallback(() => {
+    setIsPasswordVisible(!isPasswordVisible);
+  }, [isPasswordVisible]);
+
+  const MIN_LENGTH = 6;
```
Comment (plakak) [inline packages/extension-ui/src/Popup/Derive/SelectParent.tsx:152]: I would put that a bit further up - good practice is to have const on the top. And MIN_LENGTH of what? Would be nice to make sure it's MIN_PASSWORD_LENGTH :)

**Review by plakak — APPROVED** [review]

**Pull request merged**